### DISCUSSION TITLE
How can I use my style & taste of formatting for Prettier?

### DISCUSSION THREAD

**Discussion by amitkumarhq** [prettier/prettier#13174] [Q&A]
I have a habit of formatting my code like this, by adding an extra line at the end of code block, aka { } for readability purpose and also much cleaner code. I know its weird but I like the way it is, don't judge me pls :)
![image](https://user-images.githubusercontent.com/94821119/180815357-cdf74b7b-c3da-4c4a-829c-3cbee3ac07c3.png)

**Comment by JounQin** [comment]
https://prettier.io/docs/en/option-philosophy.html

https://github.com/prettier/prettier-eslint

**Reply by luxaritas** [reply]
Following up on https://github.com/prettier/eslint-config-prettier/issues/227

When using prettier-eslint without this config (since that's kinda the point), is it possible to wind up in a scenario where prettier will change something, but an eslint rule will error but not fix it, so you're in a situation where fixing it by hand just means that running prettier-eslint again will wind up bringing you right back to where you started?

**Reply by JounQin** [reply]
Again, use https://github.com/prettier/prettier-eslint instead.

**Reply by luxaritas** [reply]
Yes, that's what I'm proposing. But I'm asking if it is possible while using that to get in a situation like the one I describe (assuming after running it, I also run eslint without --fix to report non-fixable errors)

The reason I ask here rather than on that repo is that I want to know (if that is in fact an issue) if I can avoid the problem by using this config but modifying it only to disable non-fixable rules

**Reply by JounQin** [reply]
AFAIK, it's impossible. Don't waste time on formatting. 😂
